### DISCUSSION TITLE
How to limit log by directory (instead of file)

### DISCUSSION THREAD

**Discussion by paidhi** [NeogitOrg/neogit#1515] [Q&A] [answered]
Command: Log => History Simplification => Limit to files (`--`)

This opens a fuzzy file picker that lists only files.
Is there a way to allow specifying also directory names?

What I want to accomplish is the equivalent of `git log -- dirname/` instead of `git log -- dirname/somefilename`.

**Comment by CKolkey** [comment, marked as answer]
Makes sense, I can include directories in that finder so they can be chosen. Should be working as of f88d554005df47916f053399f936b90810ab7e79 :)

**Reply by paidhi** [reply]
Awesome. Thank you!

**Reply by CKolkey** [reply]
No worries, glad you asked :)
